### PR TITLE
Temp fix for skip

### PR DESCRIPTION
**Pull request by theinit01**
#32

### CODE REVIEW

**Comment by alfaarghya** [comment]
## It's great work @theinit01 , but I found some problems, please look into it

### when we use skip & limit together, skip only works on a multiple of 10, like 10, 20, 30, .......
Screenshot 2024-08-04 095242
Screenshot 2024-08-04 095309
Screenshot 2024-08-04 095331
Screenshot 2024-08-04 095351

**Comment by alfaarghya** [comment]
However, your skip query is working fine !! 😀✨

**Comment by theinit01** [comment]
@alfaarghya That is because of some backend code by Leetcode itself. They do not allow other than multiples of the limit variable coz ig they need to display on the frontend

**Comment by theinit01** [comment]
please refer to #32

File: src/Controllers/fetchProblems.ts
Comment (alfaarghya) [inline src/Controllers/fetchProblems.ts:1]: everything is fine 😀

**Comment by alfaarghya** [comment]
> @alfaarghya That is because of some backend code by Leetcode itself. They do not allow other than multiples of the limit variable coz ig they need to display on the frontend

Yeah, I understand now. Thank you for your comment 💬!

**Comment by alfaarghya** [comment]
### 🎉 Thanks for your contribution @theinit01 😊
Your code looks great, and I appreciate the thoroughness of your work. The enhancements you added are well-implemented, and I've reviewed everything. I'm happy to approve it for merging. 🚀 Thanks again for your valuable contribution 😄

**Pull request merged**